pyiode/pyiode_ws.pyx

```diff
@@ -132,32 +133,80 @@ def ws_content(pattern: str = '*', objtype: int = 6) -> List[str]:
             res[nb] = pystr(s)
             nb = nb + 1
 
-    free_tbl(cnt)
+    SCR_free_tbl(cnt)
 
     return res
 
-def ws_content_cmt(pattern: str = '*') -> List[str]:
+def ws_content(pattern: Union[str, List[str]] = '*', objtype: int = 6) -> List[str]:
+    r"""Return the names of objects of a given type, satisfying pattern specification(s). 
+    The resulting list is sorted in alphabetic order.
+
+    Parameters
+    ----------
+    pattern: string or list of strings
+        each string may contain wildcards characters like '*' or '?'.
+        Default '*', meaning "all objects".
+
+    objtype: int
+        IODE object type (0-6, 0 for comments...)
+        Default 6 for
+
+    Returns
+    -------
+    List[str]
+        List of object names
+
+    Examples
+    -------
+    >>> import iode
+    >>> iode.ws_load_cmt("../fun.cmt")               
+    317
+    >>> result = iode.ws_content("ACA*", 0)          
+    >>> print(result)                                
+    ['ACAF', 'ACAG']
+    
+    >>> result = iode.ws_content(["PIF", "ACA*"], 0) 
+    >>> print(result)                                
+    ['ACAF', 'ACAG', 'PIF']
+
+    """
+
+    if isinstance(pattern, str):
+        return(__ws_content_from_str(pattern, objtype))
+
+    elif isinstance(pattern, list):
```

Comment:
elif isinstance(pattern, list): --> elif isinstance(pattern, Iterable):

Plus, add the statement 
```python
from collections.abc import Iterable
```
at the beginning of the pyx file.